src/Moryx.Resources.Management/Resources/ResourceProxyBuilder.cs

```diff
@@ -48,11 +48,8 @@ public void PrepareBuilder()
                 return;
 
             // Check if we already created the assembly and the module builder
-#if HAVE_APPDOMAIN_DYNAMICASSEMBLY
```

Comment:
why don't we need that anymore

Reply:
The line below that does exactly this but for >.net5. It was forgotten when the net framework target was removed 🙃